src/stdio.jl

```diff
@@ -274,8 +260,8 @@ function oslibuv_flush()
 end
 
 import Base.flush
-function flush(io::IJuliaStdio)
+function flush(io::IJuliaStdio, kernel)
```

Comment:
This is a user-visible function, e.g. if the user calls `flush(stdout)`.

It would be better to store a reference to the kernel in the `IJuliaStdio` object.

Reply:
Good idea, fixed in d9acb6e.